### PR TITLE
Adding wheel configuration and release instructions

### LINKED ISSUE #1: wheel package format
Would it be possible to propose it in wheel package format on pypi ?

````
python setup.py sdist bdist_wheel --universal
twine upload dist/* 
````

### PR DESCRIPTION
**Pull request by SylvainCorlay**
Similar to those of qtawesome.

This fixes #1.

### CODE REVIEW

File: qtpy/_version.py [hunk 1]
```diff
@@ -1,2 +1,2 @@
-version_info = (0, 1, 3, 'dev')
+version_info = (0, 2, 0, 'dev')
 __version__ = '.'.join(map(str, version_info))
```
Comment (goanpeca) [inline qtpy/_version.py]: @SylvainCorlay is this 440 compliant?

**Comment by goanpeca** [comment]
@SylvainCorlay thanks my dear :-)

**Comment by SylvainCorlay** [comment]
From PEP 440:
>
> Implicit development release number
Development releases allow omiting the numeral in which case it is implicitly assumed to be 0 . The normal form for this is to include the 0 explicitly. This allows versions such as 1.2.dev which is normalized to 1.2.dev0 .

I doubt that we will ever push dev versions to pypi. The purpose of dev is mostly to differentiate the current dev version from the last release.

**Comment by goanpeca** [comment]
:+1:

**Comment by SylvainCorlay** [comment]
Well, feel free to press the green button and apply the recipe for a release. This should also fix the badges on pypi!

**Comment by ccordoba12** [comment]
@goanpeca, please upload the wheel as soon as you can :-)

**Pull request merged**